### DISCUSSION TITLE
Who is the Teddy 🐻 named after?

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#66] [Galaxy Qs] [answered]
Who is the Teddy 🐻 named after & Where is the world's largest Teddy ?

**Comment by mitchiemt11** [comment, marked as answer]
US former President Theodore Roosevelt. The largest Teddy is in Mexico.

**Reply by Sriansh-raj** [reply]
Congratulations @mitchiemt11 you just got the `Galaxy Brain` achievement. 🎉

Don't forget to create a PR and update your name in [Users_who_got_Galaxy_Brain.md](https://github.com/Sriansh-raj/GitHub-Achievements/blob/main/Users_who_got_Galaxy_Brain.md) to create a history 😃

**Comment by Denisyasyuchenya** [comment]
US former President Theodore Roosevelt. The largest Teddy is in Mexico.
